src/deadline/nuke_submitter/deadline_submitter_for_nuke.py

```diff
@@ -78,6 +78,40 @@ def _handle_step(step: dict):
         _handle_step(step)
 
 
+def _remove_gizmo_dir_from_job_template(job_template: dict[str, Any]) -> None:
```

Comment:
Could you also add a unit test for this function?

Reply:
Yep. Added the unit test for removing gizmo from job template. Thanks!